src/dvbapi.h

```diff
@@ -24,6 +24,13 @@
 #define DVBAPI_ECM_INFO 0xFFFF0003
 #define CA_SET_DESCR_MODE 0x400c6f88
 
+#define CW_ALGO_CSA 0
```

Comment:
We should use the same values as oscam otherwise there will be issues with other algos